openstack_requirements.txt

```diff
@@ -1,7 +1,7 @@
 idna==2.9
 urllib3==1.25.9
 stevedore==2.0.1
-cryptography==2.9.2
+cryptography>=3.2
```

Comment:
Please pin this to exactly version 3.2:
cryptography==3.2
This ensures that the installation will always have the right library.  Occasionally, libraries have major updates which are not backward compatible.  That breaks our functionality for customers who install after those updates.

Reply:
Done